engine/api/src/main/java/pl/touk/nussknacker/engine/api/editor/SimpleEditorType.java

```diff
@@ -0,0 +1,5 @@
+package pl.touk.nussknacker.engine.api.editor;
+
+public enum SimpleEditorType {
```

Comment:
Can't it be a `SimpleEditor` interface with two singleton implementations? We will be ready for `EnumEditor` extending `SimpleEitor` having `possibleValues` field. How you will do it with current implementation? Mayby a good thing will be to add this `EnumEditor` now?

Reply:
Why do we need to specify editor here? I mean, it doesn't make sense to have STRING_EDITOR for boolean type etc. anyway? We can detect it via type - either in BE or FE. But currently I don't see where I will be able to really **decide** which simple editor to use?

Reply:
e.g. simple editor for strings - one option: current string editor, second option: editor with list of options (currently implemented as restrictions). WDYT about this case? We are planning to replace restrictions api with current more elastic, low level api. I agree that developer can introduce but using bad editor but ... this is developer, not an end user, it is minor problem, major problem is to give him flexibility...

Reply:
... another option: editor for booleans: with true/false (for flag parameters), second option: kind of query builder.

Reply:
Real flexibility will come only with ability to add own editors :) But I get your point... Maybe let's just add TODO for some validation?

Reply:
I agree with validations. We with @PPiedel considering being even more flexible - by replacing type enum with String and adding kind of key-value map for further extensions of editor created in "user space" but we come to conclusion that it could be done when such an editors appear. WDYT?